### DISCUSSION TITLE
How to restrict channel development/creation in production environment

### DISCUSSION THREAD

**Discussion by pawangit18** [nextgenhealthcare/connect#6142] [Q&A] [answered]
Hi,
I have installed mirth (4.2) on my local system and started creating and deploying channels which is all fine.. Is it same in production environment as well, i.e channels can be created in prod env.
I was just wondering if we can disable the development activities on prod env so that prod env is used only for deployments , monitoring and other operational support activities.
thanks

**Comment by pacmano1** [comment, marked as answer]
Not really without the paid user role plugin. 

Any user that you create without this plug-in is automatically an administrator and can do anything they want.

If you don't create users, they can't login of course. Or you can restrict access to the production engine via traditional network controls.

**Comment by rjkroll** [comment]
Sounds like this can be solved with some internal business process that says 'Hey, developers, don't develop on production Mirth instances"
